day-one-challenge/tyranny-of-the-rocket/getFuelRequirements.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path')
+const text = path.join(__dirname, 'module-mass.txt')
+const massArray = fs.readFileSync(path.join(text)).toString('utf-8').split("\n");
```

Comment:
I think you could move this I/O work out of this module into it's own. This lets `getFuelRequirements` just focus on calculating fuel requirements. And, if we need to change how we get data then the `getFuelRequirements` module doesn't have to change, too.

Reply:
Sounds good! I've refactored to add the I/O work into it's own file in this commit b6faafe.